Userland/Libraries/LibWeb/HTML/ConstraintValidation.h

```diff
@@ -48,7 +48,7 @@ class ConstraintValidation {
     virtual bool is_range_overflow() const { return false; }
     virtual bool is_step_mismatch() const { return false; }
     virtual bool is_bad_input() const { return false; }
-    bool has_custom_validity_error_message() const { return !m_custom_validity_error_message.is_empty(); }
+    bool has_custom_error() const { return !m_custom_validity_error_message.is_empty(); }
```

Comment:
You've introduced this method in the previous commit, so instead of renaming it here, please pick the right name in the original commit.

Reply:
Thank you for taking a look at this. It should be resolved now.